edge-modules/TestAnalyzer/Controllers/TestResultType.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    public enum TestResultType
+    {
+        Undefined,
```

Comment:
Why define this? Does it act as a default type since it is the first?

Reply:
Undefined will be the default value of TestResultType if someone define it without assigning a value.  Therefore this will ensure values other than "undefined" is explicitly assigned.